docs/source/releases.rst

```diff
@@ -0,0 +1,64 @@
+Release History
+###############
+
+Next Release
+============
+
+Features
+--------
+- Added `AvgSignal` class. This can be used when you want to
+  run a callback on a rolling average of a ``Signal``. (#227)
```

Comment:
Do these references to PRs play nice in Sphinx? Do we want these still?

Reply:
they end up as plain text and then become links when copied to the releases page